### ISSUE TITLE
Limit length of text

### ISSUE THREAD

**Issue by garyking** [kmikiy/SpotMenu#141]
Please make it possible to set a fixed character length for the text that this app shows. So for instance, if I set a hard limit of 30 characters, then the app will truncate the total text to that length, and end it in `...`.

This would be useful, because some song titles are particularly long (`song name - featuring a, b, c - d remix`).

**Comment by 0mnius** [comment]
+1

**Comment by hazza4257** [comment]
Definitely second this. Maybe the writing could scroll across if it is over a certain length. A lot of time the menu bar is too full and since I have SpotMenu as the furthest right icon it simply disappears when the buttons on the left side take up too much space. Apart from that, great app!

**Comment by Siddhesh9146** [comment]
+1 
Really looking for this feature.

**Comment by ponugoti** [comment]
I agree that there should be an option to have the text truncate. However, I think there should be a toggle to choose between if the text will scroll or take up as much space as required. Personally, I prefer the full name of the song showing instead of a constant stream of scrolling text.
